sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 ### Features Added
 
+- Many changes to the WorkspaceConnection entity class and its associated operations.
+- Workspace Connection `list`, `get`, and `create_or_update` operations now include an optional `populate_secrets` input, which causes the operations to try making a secondary call to fill in the returned connections' credential info if possible. Only works with api key-based credentials for now.
+- Many workspace connection subtypes added. The full list of subclasses is now:
+  - `AzureBlobStoreWorkspaceConnection`
```

Comment:
I thought we agreed to call them XXXConnection instead of XXXWorkspaceConnection?

Reply:
Did we? I know they were called connections in the AI SDK, but I hadn't heard of a full renaming in this SDK. I'll double check with the PM.

Reply:
Verified with Amar that rename everything to just 'connections'